### ISSUE TITLE
Heading semantics

### ISSUE THREAD

**Issue by ashleypeacock** [melissaegg/trauma-informed-design-resources#5]
### Issue
Visual headings are not marked up as heading elements. This disadvantages assistive technology and being able to navigate easily though content.

<img width="422" alt="image" src="https://github.com/user-attachments/assets/73037c4a-3ab2-42f3-905c-f81b61dc6595">

<img width="808" alt="image" src="https://github.com/user-attachments/assets/59ac2c5e-5c56-4439-b069-3064690b3ec4">

### Recommendations

```
<div class="card-content">
            <h2 class="card-title">Gerry McGovern blog</h2> <!-- visual heading is now marked up as a heading --> 
            <p class="card-authors"><strong>By:</strong> Gerry McGovern</p>
            <p class="card-publication"></p>
            <p class="card-desc">Gerry writes about the huge, and mostly hidden, environmental impact of all things digital. He looks at things that can be done to improve the...</p>
          </div>
```

[Techniques for addressing](https://www.w3.org/WAI/WCAG21/Techniques/html/H42)

**Comment by ashleypeacock** [comment]
Updating the issue with a more helpful code recommendation.

**Comment by melissaegg** [comment]
@ksyro98 do you know how to do this one?

**Comment by ksyro98** [comment]
@melissaegg yes!
I was waiting for the other one to be merged. I will create a PR for this in a day or two.

**Comment by melissaegg** [comment]
Ah super cool, thank you! I am learning here. Appreciate you!


*Melissa Eggleston*
*she/her/hers*
UX Designer & Instructor
704-280-9080
***@***.***

Want to become (more) trauma-informed?
Visit the Trauma-Informed Technology <https://www.traumainformedtech.com/>
website.

**Comment by ksyro98** [comment]
All good @melissaegg, how git works isn't very intuitive at first.
Just created a new Pull Request for this issue.

(Once it's merged I'll work on #3.)
